app/models/billing-detail.js

```diff
@@ -73,5 +73,20 @@ export default DS.Model.extend({
 
   domainsInPlan: Ember.computed('domainAllowance', 'plan', function() {
     return this.get('domainAllowance') || DOMAIN_ALLOWANCES[this.get('plan')] || 0;
+  }),
+
+  hasCredit: Ember.computed('accountBalance', function() {
+    return this.get('accountBalance') < 0;
+  }),
+
+  accountCredit: Ember.computed('accountBalance', function() {
```

Comment:
I'm not 100% sure the order of `computed`s is guaranteed.  You should probably add `hasCredit` here as well, since it's possible `accountBalance` can trigger a new compute on this property before `hasCredit` is changed.